pkg/apis/jira/v1/types.go

```diff
@@ -1,5 +1,30 @@
 package v1
 
+const (
+	StatusNew            = "New"
+	StatusAssigned       = "ASSIGNED"
+	StatusPost           = "POST"
+	StatusInProgress     = "In Progress"
+	StatusOnQA           = "ON_QA"
+	StatusVerified       = "Verified"
+	StatusReleasePending = "Release Pending"
+	StatusClosed         = "Closed"
+	StatusModified       = "MODIFIED"
+)
+
+const (
+	ProjectKeyOCPBugs = "OCPBUGS"
+)
+
+const (
+	LabelJiraAutomator = "Regression"
```

Comment:
ComponentAutomatedRegression
Something a little more specific would be good here, doesn't have to be this, but clarity that this came from our code.